ocs_ci/ocs/utils.py

```diff
@@ -925,45 +956,115 @@ def run_must_gather(log_dir_path, image, command=None, cluster_config=None):
             cluster_config=cluster_config,
         )
     except CommandFailed as ex:
-        log.error(get_helper_pods_output())
         log.error(
             f"Failed during must gather logs! Error: {ex}"
             f"Must-Gather Output: {mg_output}"
         )
+        export_mg_pods_logs(log_dir_path=log_dir_path, mg_output=mg_output)
```

Comment:
BTW. is the `mg_output` variable really defined if there were raised some exception in the `exec_oc_cmd` method?

Reply:
I deleted `mg_output` arg